src/embedded.ts

```diff
@@ -128,6 +133,9 @@ export class RoomWidgetClient extends MatrixClient {
     private lifecycle?: AbortController;
     private syncState: SyncState | null = null;
 
+    private pendingSendingEventsTxId: { type: string; id: string | undefined; txId: string }[] = [];
+    private pendingEventsEmitter = new TypedEventEmitter<PendingEvent.PendingEventsChanged, PendingEventHandlerMap>();
```

Comment:
Can this just be `eventEmitter` as it is more than likely to get reused later in its life and we'd rather not have the next use spin up a 2nd.